libafl_bolts/src/lib.rs

```diff
@@ -369,6 +371,12 @@ impl Error {
         Error::EmptyOptional(arg.into(), ErrorBacktrace::new())
     }
 
+    /// The `Input` was invalid
+    #[must_use]
+    pub fn invalid_input() -> Self {
+        Error::InvalidInput(ErrorBacktrace::new())
```

Comment:
Stupid idea, @tokatoka, should we _not_ add ErrorBacktrace here, to not add any overhead in this case, and then make backtraces default?

Reply:
(Then again, this isn't in the hot path of most fuzzers, so probably not worth doing/little overhead)

Reply:
i don't think it is worth it